frontend/javascripts/oxalis/controller/combinations/move_handlers.ts

```diff
@@ -45,9 +47,10 @@ export const moveW = (deltaW: number, oneSlide: boolean): void => {
   }
 
   if (oneSlide) {
-    const logZoomStep = getRequestLogZoomStep(Store.getState());
+    const activeMagIndices = getActiveMagIndicesForLayers(Store.getState());
+    const finestMagIdx = _.min(values(activeMagIndices)) || 0;
     const wDim = Dimensions.getIndices(activeViewport)[2];
-    const wStep = getResolutions(Store.getState().dataset)[logZoomStep][wDim];
+    const wStep = getResolutions(Store.getState().dataset)[finestMagIdx][wDim];
```

Comment:
> On a different note, maybe it's more intuitive if this only takes into account layers that are currently visible? Otherwise, if the layer with the finest resolution is not visible, the step might be too small and there will be no change in the viewports.

This is a good point :+1: 

> I'm not sure I understand how this continues to work 
My current assumption is that there are layers with varying scales and, therefore, the one with the finest resolution not necessarily is the one with the smallest voxel size.
Instead, it's likely the case that the resolutions refer to a base scale - in which case it does make sense again.

Let's discuss this in-person :)